integration/extended_job_test.go

```diff
@@ -194,33 +195,34 @@ var _ = Describe("ExtendedJob", func() {
 			pod := env.LabeledPod("nomatch", testLabels("key", "nomatch"))
 			tearDown, err := env.CreatePod(env.Namespace, pod)
 			Expect(err).NotTo(HaveOccurred())
-			defer tearDown()
+			defer func(tdf environment.TearDownFunc) { Expect(tdf()).To(Succeed()) }(tearDown)
 			err = env.WaitForPods(env.Namespace, "test=true")
 			Expect(err).NotTo(HaveOccurred(), "error waiting for pods")
 
 			By("creating a first extended job")
 			ej := *env.DefaultExtendedJob("extendedjob")
 			_, tearDown, err = env.CreateExtendedJob(env.Namespace, ej)
 			Expect(err).NotTo(HaveOccurred())
-			defer tearDown()
+			defer func(tdf environment.TearDownFunc) { Expect(tdf()).To(Succeed()) }(tearDown)
 
 			By("triggering another reconciliation")
 			pod = env.LabeledPod("foo", testLabels("key", "value"))
 			tearDown, err = env.CreatePod(env.Namespace, pod)
 			Expect(err).NotTo(HaveOccurred())
-			defer tearDown()
+			defer func(tdf environment.TearDownFunc) { Expect(tdf()).To(Succeed()) }(tearDown)
 
 			By("waiting for the job")
 			_, err = env.CollectJobs(env.Namespace, "extendedjob=true", 1)
 			Expect(err).NotTo(HaveOccurred(), "error waiting for jobs from extendedjob")
 
 			// Why does the ExtendedJob deletion not trigger the job deletion?
```

Comment:
This comment can probably be removed now?

Reply:
👍